java/jdbc-spring-jdbcclient-oracle/src/main/java/com/oracle/jdbc/newclient/EmployeeMessages.java

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright (c) 2024, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.jdbc.newclient;
+
+public enum EmployeeMessages {
+
+  EMPLOYEE_CREATION_FAILED(
+      "Employee creation failed: "), EMPLOYEE_UPDATE_FAILED(
```

Comment:
Please fix the format

Reply:
The indentation is double-checked for all classes. Please note and remember the enforced format for String and enum data types as expected, per the IDE conventions we configured together. @Kuassim please review, thanks